File: CTDK_Python/140/_140.py
from math import sqrt

a = float(input("Nhap a: "))
b = float(input("Nhap b: "))
c = float(input("Nhap c: "))

delta = b * b - 4 * a * c
if (delta <= 0):
    if (delta == 0):
        x0 = -b / (2*a)
        print("Nghiem kep: ", x0)
    else:
        print("Vo nghiem!!!")
else:
    x1 = (-b + sqrt(delta)) / (2 * a)
    x2 = (-b - sqrt(delta)) / (2 * a)
    print("Nghiem thu nhat: ", x1)
    print("Nghiem thu hai: ", x2)

File: CTDK/140/140.cpp
#include <iostream>
using namespace std;

int main()
{
	float a;
	cout << "Nhap a: ";
	cin >> a;

	float b;
	cout << "Nhap b: ";
	cin >> b;

	float c;
	cout << "Nhap c: ";
	cin >> c;

	float delta = b * b - 4 * a * c;
	if (delta <= 0)
	{
		if (delta == 0)
		{
			float x0 = -b / (2 * a);
			cout << "x0 = " << x0 << endl;
		}
		else
		{
			cout << "Phuong trinh vo nghiem!!!!!" << endl;
		}
	}
	else
	{
		float x1 = (-b + sqrt(delta)) / (2 * a);
		float x2 = (-b - sqrt(delta)) / (2 * a);
		cout << "x1 = " << x1 << endl;
		cout << "x2 = " << x2 << endl;
	}
	return 0;
}